src/app/operators/dialogs/raster-mask/raster-mask.component.html

```diff
@@ -1,8 +1,14 @@
 <wave-sidenav-header>Raster Mask</wave-sidenav-header>
 <form [formGroup]="form" (ngSubmit)="add()" fxLayout="column">
+    <wave-dialog-help>
+        The <em>Raster Mask</em> modifies an input raster layer by applying a second mask raster layer.
```

Comment:
```suggestion
        The <em>Raster Mask</em> modifies an input raster layer by applying another raster layer as a mask .
```
there is only one mask